service-api/composer.json

```diff
@@ -7,7 +7,7 @@
     "repositories": [
     ],
     "require": {
-        "php": ">=8.1 <8.2.0",
+        "php": ">=8.1 <8.2.2",
```

Comment:
We don't want any 8.2 PHP versions in composer.json files, except for in service-admin. This causes a potential issue if a PHP library is compatible with 8.2 but not 8.1.

Reply:
OK sorted